.github/workflows/build-and-deploy.yml

```diff
@@ -9,7 +9,7 @@ on:
         default: 'all'
         type: choice
         options:
-#          - all
```

Comment:
I would just remove this and not comment it out. Why are the two components called out here explicitly?